package.json

```diff
@@ -23,13 +26,5 @@
   },
   "dependencies": {
     "cypress": "^9.6.1"
```

Comment:
Cypress shouldn't be a dependency, only a devDependency, also, did you erase the scripts?